### DISCUSSION TITLE
Dropdown item activation

### DISCUSSION THREAD

**Discussion by amliebsch** [Tavenem/Blazor.Framework#9] [Q&A]
First, thank you for your work on this framework.  I've found it really useful.

My question is how can I make the child items in a Dropdown container clickable?  It seems that no matter what I do, the click event is consumed by the Dropdown and I am not able to actually trigger any actions with the child items.

For example:

```
<Dropdown Text="..."  >
    <button class="btn" @onclick=Test1>Test 1</button>    
</Dropdown>

@code {
private async void Test1()
{
    System.Diagnostics.Debug.WriteLine("Item one clicked");
    Console.WriteLine("Item one clicked");
}
```

The code in the Test1 method never runs.

**Comment by WilStead** [comment]
Thank you for bringing this to my attention!

This was definitely a bug, introduced sometime in the 2.x timeframe.

I believe I have resolved it in v2.14, just released. Child elements of dropdowns should now respond properly to events.
